server/public/model/websocket_message.go

```diff
@@ -9,87 +9,97 @@ import (
 	"strconv"
 )
 
+type WebsocketEventType string
+
+func ToStringArray(arr []WebsocketEventType) StringArray {
```

Comment:
0/5 but it might be a bit premature to promote this as a public API. I'm happy to keep this as a private function inside the shared_channel_notifier.go file until we see usage from other places.

Reply:
The `ToStringArray` method is very specific to the type `WebsocketEventType`, I feel it should be closer to where the type is defined, which is why I had also created a `ToStringArray` [method in previous commit](https://github.com/mattermost/mattermost/pull/25195/commits/a76593d91f5dc5199662f80c598cd55ca6040091#r1391171325)

what do you think @agnivade ?

Reply:
Yes it's related to that type, but it's only used once. We can change the method to a function and keep that private. A good rule of thumb is to start with private helper functions, and then move them to a common place if used from multiple places.

Reply:
make sense, have moved the method and made it private